cmake/PACE_Docs.cmake

```diff
@@ -0,0 +1,65 @@
+# Build documentation
+
+set(DOCS_DIR "${Horace_ROOT}/documentation/user_docs" CACHE STRING "Directory containing user documentation")
+
+find_program(sphinx-build NAMES sphinx-build)
+find_program(pdflatex NAMES pdflatex)
+find_program(latexmk NAMES latexmk)
+if (sphinx-build)
+  if (UNIX)
+    add_custom_target(denullify-docs
```

Comment:
Is it possible to perform some of these processes using `add_custom_command`? I think we're exposing more build targets than we need to here. I think we only really need the `docs` and `manual` processes to be targets